src/WebApi/Controllers/UsersController.cs

```diff
@@ -356,6 +356,18 @@ public Task<ActionResult<Result<CharacterLimitationsViewModel>>> GetCharacterLim
         }));
     }
 
+    /// <summary>
+    /// Resets a character Rating.
+    /// </summary>
+    /// <param name="userId">Owner of Character userId.</param>
+    /// <param name="id">Character id.</param>
+    /// <response code="200">Character rating reseted.</response>
+    /// <response code="400">Bad Request.</response>
+    /// <response code="404">Character not found.</response>
+    [HttpPut("{userId}/characters/{id}/retire")]
```

Comment:
This endpoint should be restricted to admins only

Reply:
done